src/content/tools/dart-doc.md

```diff
@@ -71,7 +71,7 @@ check out [dart.dev/go/dartdoc-options-file][dartdoc-options].
 TODO: Document the long-term supported options here.
 {% endcomment -%}
 
-[dartdoc-options]: https://dart.dev/go/dartdoc-options-file
+[dartdoc-options]: {{site.url}}/go/dartdoc-options-file
```

Comment:
I'd prefer we avoid `{{site.url}}` in links if possible, but the full link is necessary for the go-links and other redirects like this. Can you create a few new config variables like `{{site.redirect}}` and `{{site.redirect.go}}` for use cases like this?

Reply:
added:
```yaml
redirect:
  go: https://dart.dev/go
```